src/components/inputs/LinkInput.vue

```diff
@@ -42,9 +57,45 @@ const options = ref(plugin.value.data.getComponentsByType(
 const iconName = ref(plugin.value.data.definitions.components.find(
   ({ type }) => type === attribute.value.definition.linkRef,
 ).icon);
+const defaultValues = ref(plugin.value.data.getComponentsByType(
+  attribute.value.definition.linkRef,
+).map(({ id }) => id));
+const variables = ref(plugin.value.data.variables || []);
+
+/**
+ * Initialize the options for the select.
+ */
+function initOptions() {
```

Comment:
already used 3times. I would rather put this in some kind of utility file and import this function in the component